doc/USAGE.md

```diff
@@ -0,0 +1,187 @@
+- The Revault ceremony
+    - Revaultd supposes that you've already done a proper ceremony with all the stakeholders and managers, where you created the keys in a secure way and exchanged them.
+    - If you're just exploring revault you probably don't have time to do a proper ceremony: let's recreate something similar.
+    - First of all, it's important to clarify how many stakeholders and managers you'll have: you must have at least 2 stakeholders and 1 manager, and each participant gets its own computer. You must decide before starting who is a manager, who is a stakeholder, who is both. If you're by yourself but you still want to try out Revault, you can get around this by running multiple revaultd instances with multiple configuration files.
+    - Each entity in the company will have a pair of BIP32 keys: generating them is the first step in the ceremony. The ceremony is not fully specified yet, but it will instruct managers and stakeholders on how to securily create and store the keys. Since we're on regtest anyway, we don't really care much about security in this specific example: just create the keys on bip32.org or, if you don't know how to do it, just use the ones provided in the [example config](contrib/config_regtest.toml).
+
+## Downloading the repositories
+Let's start by creating a dedicated folder - this way if you don't like Revault it will be easier to erase everything :D
+```
+mkdir revault_tutorial
+cd revault_tutorial
+```
+Download all the needed repositories: `cosignerd`, `coordinatord`, `revaultd` and optionally, `revault-gui`:
+```
+git clone https://github.com/revault/coordinatord
+git clone https://github.com/revault/cosignerd
+git clone https://github.com/revault/revaultd
+git clone https://github.com/revault/revault-gui
+```
+
+You working directory should look like:
+```
+.
+├── coordinatord
+├── cosignerd
+├── revaultd
+├── revault-gui
+```
+
+## Let's get started!
+### 1. Spinning up Bitcoin Core
+In this tutorial we'll use the regtest network. We'll use a custom data directory inside the `revault_tutorial`.
+```
+mkdir revault_core_data
+```
+
+Start a Bitcoin Core node using
+```
+bitcoind -regtest -daemon -datadir=$(pwd)/revault_core_data
+```
+
+### 2. Starting the coordinator
+Cd into the coordinator:
+```
+cd coordinatord
+```
+TBD: docker script?
+
+After starting the coordinator, please keep note of the host and the noise key used, we'll need them later.
+
+Go back to the parent directory for the next step
+```
+cd ..
+```
+
+### 3. Configuring the cosigner
+Cd into the cosignerd:
+```
+cd cosignerd
+```
+
+You can find an [example config](https://github.com/revault/cosignerd/tree/master/contrib/config_regtest.toml) to begin with. Copy it to `./our_modified_config.toml`
+```
+cp contrib/config_regtest.toml our_modified_config.toml
+```
+We'll need to modify it a bit:
+- Update the manager xpubs with the xpubs generated at the ceremony.
```

Comment:
No need for the managers xpubs on the cosigning servers

Reply:
LOL

I really think that we should display a warning when there's stupid shit in the configuration, I've kept them there for weeks